src/components/Question.tsx

```diff
@@ -0,0 +1,22 @@
+import { MDXProvider } from "@mdx-js/react";
+
+import MDXComponents from "./MDXComponents";
+
+type Props = {
+  children: React.ReactNode;
+  author?: string;
+  authorLink?: string;
+  date: string;
+};
+export default function Question({
+  children,
+  author,
+  authorLink,
+  date,
+}: Props) {
+  return (
+    <div>
```

Comment:
Div replace garda hunxa with `<></>` fragment.

Reply:
just a proof of concept for now. Here we can style question as we want and same with answer component.